src/scss/settings/_variables.scss

```
@@ -97,28 +97,22 @@ $responsive-gutters           : true !default; // Disable this if you don't want
 $use-legacy-grid              : true !default;
 
 
-// Spacing (Vertical / Horizontal)
-// ==========================================================================
-$spacing: 8px; // only for use in map below – not for in components
-
 // spacing map
-// Can access levels by using the spacing function > spacing() or spacing(x5) for example
+// Can access levels by using the spacing function > spacing() or spacing(a) for example
 $spacing: (
-    x0.5 : $spacing / 2,
-    base : $spacing,
-    x1.5 : $spacing * 1.5,
-    x2   : $spacing * 2,
-    x3   : $spacing * 3,
-    x4   : $spacing * 4,
-    x5   : $spacing * 5,
-    x6   : $spacing * 6,
-    x7   : $spacing * 7,
-    x8   : $spacing * 8,
-    x9   : $spacing * 9,
-    x10   : $spacing * 10
+    a    : $spacing-a,
```

I know it'll be potentially flakey to do this, but would it be handy to potentially comment here what the values are (as I would think they'll rarely change in the future). Just thinking it may save people some time looking up the variables for these values – what do you think?

```suggestion
    a    : $spacing-a, // 4px
```

---

Makes sense, ill add that in.

---

Fixed.